### PR TITLE
Fixed Typos in README.md

### PR DESCRIPTION
**Pull request by Gurveen-codes**
There were some typos in Readme file
typo-fix

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ yarn start
 
 > Ensure you are not using Power shell
 
-6. 🎉 Open your browser and go to http://localhost:8000/ or http://localhost:8000/___graphql
+6. 🎉 Open your browser and go to http://localhost:8000/ or http://localhost:8000/\_\_\_graphql
```
Comment (ming-tsai) [inline README.md:47]: Could you rollback this line?
Because the link was corrected

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @Gurveen-codes,
Nice work, only one comment for the link

File: README.md (hunk 1 above)
Comment (Gurveen-codes) [reply README.md]: @ming-tsai I corrected that 👍🏽

**Review by ming-tsai — APPROVED** [review]
Nice work @Gurveen-codes, and welcome to our team
> Invitation the [link](https://github.com/MakeContributions)

**Pull request merged**